main.scss

```diff
@@ -5,174 +5,194 @@
 @import "o-grid/main";
 @import "o-hoverable/main";
 
-$o-ft-footer-spacing-unit: 20px;
+/// Silent mode
+/// @type Bool
+$o-footer-is-silent: true !default;
 
-@include oFontsInclude(BentonSans);
+$o-footer-spacing-unit: 20px;
 
 // TODO: Remove this when a default shared brand color is defined in o-colors
-@include oColorsSetUseCase(o-ft-footer-brand, border, claret);
-
-@include oColorsSetUseCase(o-ft-footer, background, grey-tint5);
-@include oColorsSetUseCase(o-ft-footer, text, white);
-@include oColorsSetUseCase(o-ft-footer__pearson, background, grey-tint4);
-@include oColorsSetUseCase(o-ft-footer__divider, border, white);
-@include oColorsSetUseCase(o-ft-footer-item, text, white);
-
-.o-ft-footer {
-	@include oColorsFor(o-ft-footer, text background);
-	@include oFtTypographyFontSize(14);
-
-	margin-top: 40px;
-	padding: 0;
-	border-top: 5px solid oColorsGetColorFor(o-ft-footer-brand brand, border);
-	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
-	line-height: $o-ft-footer-spacing-unit;
-
-	> nav {
-		margin-top: $o-ft-footer-spacing-unit;
-		margin-bottom: $o-ft-footer-spacing-unit;
-		padding-bottom: $o-ft-footer-spacing-unit;
-		border-bottom: oColorsGetColorFor(o-ft-footer__divider, border) dotted 1px;
-	}
+@include oColorsSetUseCase(o-footer-brand, border, claret);
+
+@include oColorsSetUseCase(o-footer, background, grey-tint5);
+@include oColorsSetUseCase(o-footer, text, white);
+@include oColorsSetUseCase(o-footer__pearson, background, grey-tint4);
+@include oColorsSetUseCase(o-footer__divider, border, white);
+@include oColorsSetUseCase(o-footer-item, text, white);
+
+@if $o-footer-is-silent == false {
+	@include oFontsInclude(BentonSans);
 
-	ul {
-		list-style-type: none;
+	.o-footer {
```

Comment:
seems reasonable to me

On 21 January 2015 at 09:45, Rhys Evans <notifications@github.com> wrote:

> In main.scss
> <https://github.com/Financial-Times/o-ft-footer/pull/23#discussion_r23287701>
> :
>
> >
> > -	ul {
> > -		list-style-type: none;
> > +	.o-footer {
>
> Can o-footer colours be overwritten after o-footer has set them? I ask
> because I wonder if there's a use case for the footer but in a different
> colour. Header is a nice neutral grey but the footer, being a shade of
> pink, may clash on some microsites?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-ft-footer/pull/23/files#r23287701>.
>



-- 

------------------------------
*Paul O'Neill*
Senior Interaction Designer
FT.com

+44 (0)20 7873 3069
Number One Southwark Bridge, London SE1 9HL
Skype: pauloneillft
Blog: http://bit.ly/MyNeoBlog

<https://www.facebook.com/financialtimes> <https://twitter.com/FT>
<http://www.linkedin.com/company/financial-times>
<https://plus.google.com/+FinancialTimes/posts>
<http://www.youtube.com/user/FinancialTimesVideos>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.